dashboard/app/controllers/projects_controller.rb

```diff
@@ -15,6 +15,9 @@ class ProjectsController < ApplicationController
     playlab: {
       name: 'New Play Lab Project'
     },
+    starwars: {
+      name: 'New Star Wars Project'
```

Comment:
(These level names are only for editing the project template on Level Builder — they're not visible anywhere publicly.)